test/e2e/apimachinery/resource_quota.go

```diff
@@ -1992,7 +1993,7 @@ func newTestReplicationControllerForQuota(name, image string, replicas int32) *v
 			Name: name,
 		},
 		Spec: v1.ReplicationControllerSpec{
-			Replicas: func(i int32) *int32 { return &i }(replicas),
+			Replicas: pointer.Int32Ptr(replicas),
```

Comment:
I just remembered that `Int32Ptr` [is deprecated](https://github.com/kubernetes/utils/blob/a5ecb0141aa5b1b224e745aca203afdfd22d8b3a/pointer/pointer.go#L79-L82). Please use `Int32` instead.

Reply:
Replaced `Int32Ptr` with `Int32`.